ietf-scitt-charter.md

```diff
@@ -4,9 +4,7 @@ Over the years, rapid technological advancements have motivated organizations to
 While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size, and complexity of supply chains has led to more frequent and sophisticated supply chain attacks.
 The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate.
 
-The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of supply chain systems and assisting implementers in securing deployments.
-For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in an append-only transparent registry.
-Therefore, providing an individual using the system with confidence that it will behave as and when expected, consistently and without deviation.
+SCITT forms a set of interoperable building blocks that will allow implementers to build integrity and accountability into supply chain systems to help assure trustworthy operation. For example, a public computer interface system could report its software composition that can then be compared against known software compositions (and certifications?) for such a device thereby giving confidence that the system is running the software expected and has not been modified, either by attack or accident, in the supply chain.
```

Comment:
To address Orie's comment, how about this:

> **The output of the SCITT WG is** a set of interoperable building blocks that will allow implementers to build integrity and accountability into supply chain systems to help assure trustworthy operation.

Instead of this...

> **SCITT forms** a set of interoperable building blocks that will allow implementers to build integrity and accountability into supply chain systems to help assure trustworthy operation.

Otherwise, I agree with this change.

Reply:
I'm good with that proposed language.


@henkbirkholz is this language ok for a charter? 

I like the ideal of being a bit concrete about what we intend to deliver.

Reply:
Only a few editorial requests:

* the 2nd sentence now is pretty long and should be split
* "(and certifications?)" cannot stay, either remove it or incorporate it as "or certifications"

Otherwise I am okay with the suggested change, the core of the message remains unchanged.

Reply:
All addressed in update #2. @henkbirkholz are you happy to merge?

Reply:
I am happy with Kay's proposal and will incorporate!